src/modules/esl-utils/dom/script.ts

```diff
@@ -1,23 +1,22 @@
 /**
- * Creates new async script tag by id and src
+ * Creates new script element with specified attributes
  */
-const createAsyncScript = (id: string, src: string): HTMLScriptElement => {
+const createScript = (attrs: Record<string, string | boolean>): HTMLScriptElement => {
   const script = document.createElement('script');
-  script.id = id;
-  script.async = true;
```

Comment:
Note: the `async = true` - legacy operation here and mostly nominal (mb except IE). All currently supported browsers always load manually added scripts asynchronously.

Reply:
ok, I will remove this